scapy/packet.py

```diff
@@ -223,6 +223,26 @@ def __init__(self,
         Optional[bytes],
     ]
 
+    @property
+    def comment(self):
+        # type: () -> Optional[bytes]
+        """Get the comment of the packet"""
+        if self.comments is not None and len(self.comments) > 0:
+            return self.comments[-1]
```

Comment:
I will expect this function to return the first element.
```suggestion
            return self.comments[0]
```

Reply:
I fixed it 👍